go.mod

```diff
@@ -30,5 +30,11 @@ exclude (
 	github.com/coreos/etcd v3.3.12+incompatible
 	github.com/coreos/etcd v3.3.13+incompatible
 	github.com/coreos/etcd v3.3.15+incompatible
+	github.com/coreos/etcd v3.3.16+incompatible
```

Comment:
Lol this seems a bit excessive

Reply:
Haha, yeah. There is a `replace` directive in go mod. And while it works to force 3.3.10 => 3.3.22 in a single line, srcclr doesn't appear to respect that and still flags the original version 3.3.10 which has even higher severity warnings.